lib/servers/evPump/evPumpServer.py

```diff
@@ -0,0 +1,227 @@
+    @setting(2, 'toggle shutter', value = 'b')
+    def toggle_shutter(self, c, value):
+        if value:
+            yield self.ser.write_line('SHT:1')
+        else:
+            yield self.ser.write_line('SHT:0')
+            
+    @setting(3, 'set power', value = 'v[W]')
+    def set_power(self, c, value):
+        value = str(value['W'])
+        yield self.ser.write_line('P:' + value)
+        
+    @setting(4, 'read power', returns = 'v[W]')
+    def read_power(self,c):
+        yield None
+        returnValue(self.power)
+        
+    @setting(5, 'set current', value = 'v[A]')
+    def set_current(self, c, value):
+        value = str(value['A'])
+        yield self.ser.write_line('C1:' + value)
+        
+    @setting(6, 'read current', returns = 'v[A]')
+    def read_current(self,c):
+        yield None
+        returnValue(self.current)
+
+    @setting(7, 'diode status', returns = 'b')
+    def diode_status(self,c):
+        yield self.ser.write_line('?D')
+        value = yield self.ser.read_line()
+        value = bool(float(value))
+        returnValue(value)
+        
+    @setting(8, 'system status', returns = 's')
+    def system_status(self,c):
+        yield None
+        returnValue(self.status)
+        
+    @setting(9, 'get power setpoint', returns = 'v[W]')
+    def get_power_setpoint(self, c):
+        yield self.ser.write_line('?PSET')
+        value = yield self.ser.read_line()
+        value = U(float(value), 'W')
+        returnValue(value)
+        
+    @setting(15, 'get current setpoint', returns = 'v[A]')
+    def get_current_setpoint(self, c):
+        yield self.ser.write_line('?CS1')
+        value = yield self.ser.read_line()
+        if value:
+            value = U(float(value), 'A')
+        else:
+            value = U(0.0, 'A')
+        returnValue(value)
+        
+    @setting(10, 'get shutter status', returns = 'b')
+    def get_shutter_status(self, c):
+        yield self.ser.write_line('?SHT')
+        value = yield self.ser.read_line()
+        value = bool(float(value))
+        returnValue(value)
+    
+    @setting(11, 'set control mode', mode = 's')
+    def set_control_mode(self, c, mode):
+        if mode == 'current':
+            yield self.ser.write_line('M:0')
+        elif mode == 'power':
+            yield self.ser.write_line('M:1')            
+        else:
+            yield None
+            
+    @setting(12, 'get control mode', returns = 's')
+    def get_control_mode(self, c):
+        yield self.ser.write_line('?M')
+        value = yield self.ser.read_line()
+        if value == '0':
+            value = 'current'
+        elif value == '1':
+            value = 'power'
+        else:
+            value = None
+        returnValue(value)
+        
+    @setting(13, 'read temperature', returns = 'v[degC]')
+    def read_temperature(self,c):
+        yield None
+        returnValue(self.temperature)
+        
+    @setting(14, 'get diode current limit', returns = 'v[A]')
```

Comment:
I think at least in my own case I would prefer not to have anything in the string because I simply did not know what that string does. I would prefer to have just a function name.